code/modules/cargo/packs/engineering.dm

```diff
@@ -57,12 +56,26 @@
 				)
 	crate_name = "engineering gear crate"
 
+/datum/supply_pack/engineering/cones
+	name = "Engineering Hat Crate"
+	desc = "A complete set of headwear to fit the heads of an entire engineering crew. Includes 6 cones."
+	cost = CARGO_CRATE_VALUE * 1.5
+	contains = list(/obj/item/clothing/head/cone = 5)
+	crate_name = "engineering hat crate"
+
+/datum/supply_pack/engineering/radios
+	name = "Telecommunications Starter Pack crate"
+	desc = "Contains everything oyu need to get a basic stationwide radio network established!"
```

Comment:
i have no idea what "oyu" is, but it scares me and id beg you to re-write this summonation word of the elder demons